### PR TITLE
[disaster] remove NA from top event tiles

### PR DESCRIPTION
**Pull request by chejennifer**
Screenshot 2023-06-05 at 2 18 12 PM

### CODE REVIEW

File: static/js/components/tiles/top_event_tile.tsx [hunk 1]
```diff
@@ -132,7 +132,13 @@ export const TopEventTile = memo(function TopEventTile(
                 <tr>
                   <td></td>
                   {showNameColumn && <td>Name</td>}
-                  {showPlaceColumn && <td>{props.enclosedPlaceType}</td>}
+                  {showPlaceColumn && (
+                    <td>
+                      {Object.keys(eventPlaces).length < topEvents.length
```
Comment (juliawu) [inline static/js/components/tiles/top_event_tile.tsx:137]: For my own understanding, why compare number of event places with number of top events?

**Review by juliawu — APPROVED** [review]
Thanks for the changes!

File: static/js/components/tiles/top_event_tile.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/components/tiles/top_event_tile.tsx:137]: eventPlaces has a key value pair for every event that has a place found for it. Therefore, if there are less eventPlaces entries than number of events, that means at least one event is missing the place

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**